types/stylelint/index.d.ts

```diff
@@ -371,7 +371,7 @@ declare namespace stylelint {
 			true,
 			{
 				ignoreShorthands: OneOrMany<StringOrRegex>;
-				ignoreLonghands: string[];
+				ignoreLonghands: OneOrMany<string>;
```

Comment:
[ask] Is it not easy to support also regex?

```suggestion
				ignoreLonghands: OneOrMany<StringOrRegex>;
```

Reply:
It could be done in a separate PR. I cannot find any cases where you would want to use regexes though.
i.e. it's either `"text-decoration-thickness"` or `"background-size", "background-origin", "background-clip"` or `"transition-behavior"`

If someone requests it, we will add it.

Reply:
I see. Keeping it makes sense for now. 👍🏼